src/style.css

```
@@ -1,3 +1,70 @@
 body {
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: "Roboto", sans-serif;
+  font-weight: normal;
+  font-size: 12px;
+  --link-height: 60px;
+  --color-is-active: #00acdc;
+  --color--nav: #000;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  text-transform: uppercase;
+}
+
+.logo__img {
+  display: block;
+}
+
+.nav {
+  display: block;
+}
+
+.nav__list {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  list-style-type: none;
+  padding: 0;
+  margin: 0;
+}
+
+.nav__item:not(:last-child) {
+  margin-right: 20px;
+}
+
+.nav__link {
+  position: relative;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  height: var(--link-height);
```

U did same problem. What happens if I set a font size 3 times larger than the one I have now? It will all break, so it's better to use padding that simply "wraps the content"
```suggestion
  padding: 23px 0
```